demisto_sdk/commands/json_to_outputs/json_to_outputs.py

```diff
@@ -164,6 +164,10 @@ def parse_json(data, command_name, prefix, verbose=False, interactive=False):
 
         raise ValueError('Invalid input JSON')
 
+    # if data is a list of dictionaries [{'a': 'b', 'c': 'd'}, {'e': 'f'}] -> {'a': 'b', 'c': 'd', 'e': 'f'}
+    if isinstance(data, list):
+        data = {k: v for d in data for k, v in d.items()}
```

Comment:
what if two dicts in the list have the same keys?

Reply:
The second dictionary overwrites the first
![Screen Shot 2021-07-04 at 13 46 43](https://user-images.githubusercontent.com/78307768/124382220-f8c58500-dcce-11eb-8e9b-82e31d49417a.png)